contracts/axelar-interchain-token-service/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+#![no_std]
+
+mod error;
```

Comment:
```suggestion
pub mod error;
```
allows use in integration tests and for consumers